multicluster/cmd/multicluster-controller/controller.go

```diff
@@ -168,12 +168,15 @@ func run(o *Options) error {
 		setupLog.Error(err, "unable to create controller", "controller", "ResourceExport")
 		os.Exit(1)
 	}
-	if err = (&multiclustercontrollers.ResourceImportReconciler{
-		Client: mgr.GetClient(),
-		Scheme: mgr.GetScheme(),
-	}).SetupWithManager(mgr); err != nil {
+	resImportReconciler := multiclustercontrollers.NewResourceImportReconciler(
+		mgr.GetClient(),
+		mgr.GetScheme(),
+		localMgr,
+		remoteMgr,
```

Comment:
I think common area doesn't support initialize both localMgr and remoteMgr now,  it will only create remoteMgr when it's member cluster and create localMgr when it's leader cluster. need @aravindakidambi to double check and clarify this.

Reply:
I have refined `clusterset_controller.go` so it can support to run as both leader and member in a new commit